wormhole-connect/src/utils/gasEstimates.ts

```diff
@@ -253,6 +253,16 @@ const getGasFeeFallback = async (
       );
     const gasFees = BigNumber.from(gasEst).mul(gasPrice);
     return toFixedDecimals(utils.formatEther(gasFees), 6);
+  } else if (route === Route.CCTPRelay) {
+    const gasEst = gasEstimates.sendCCTPWithRelay;
```

Comment:
Maybe we can clean up the gas estimate logic a bit by creating a mapping of route to gas estimates.  Something like
```
const gasEstimates = {
  [Route.BRIDGE]: {
    send: ...,
    sendNative: ...,
    claim: ...,
  },
  [Route.RELAY]: {
    send: ...,
    sendNative: ...,
  },
  etc...
}
```